docs/tech-reference/export-distributor.rst

```diff
@@ -62,6 +62,9 @@ Optional Configuration Parameters
  ISO images and published over HTTP or HTTPS. Instead, they are written to the export directory.
  This option is useful if exporting to an external hard drive, for example.
 
+``relative_url``
```

Comment:
i manually tested the case where i provide the relative url but it still exports under repo's name and ignores the one that i provide